client/landing/gutenboarding/onboarding-block/features/data.tsx

```diff
@@ -0,0 +1,161 @@
+				/>
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M4 2.27539C2.78 3.32215 2 5.11415 2 6.8265C2 9.34099 3.79 11.3776 6 11.3776C6.387 11.3776 6.753 11.2866 7.093 11.1364C7.686 11.9761 8.28 12.8158 8.9 13.6304L10.978 11.2638C10.264 10.5584 9.528 9.8803 8.788 9.20446C8.92 8.81989 9 8.40346 9 7.96428C9 6.07898 7.657 4.55095 6 4.55095C4.69 4.55095 4 3.23681 4 2.27539ZM15.492 13.0934L14.152 14.618C16.08 16.7627 17.905 19.0258 19.6 21.4185L20.75 23.0398L19.25 24.7465L17.825 23.4403C15.722 21.5095 13.733 19.4331 11.848 17.2394L5 25.031L3 22.7554L13.493 10.8167C13.188 10.1261 13 9.35806 13 8.53317C13 5.70579 15.015 3.41317 17.5 3.41317C18.225 3.41317 18.9 3.62707 19.507 3.97409L16 7.96428L18 10.2398L21.507 6.24965C21.812 6.94028 22 7.70828 22 8.53317C22 11.3605 19.985 13.6532 17.5 13.6532C16.775 13.6532 16.099 13.4393 15.492 13.0934Z"
+				/>
+			</SVG>
+		),
+	},
+	[ FEATURE_AD_FREE ]: {
+		id: FEATURE_AD_FREE,
+		name: translate( 'Ad-free' ) as string,
+		description: translate( 'Remove advertisements and own your brand.' ) as string,
+		minSupportedPlan: PLAN_PERSONAL,
+		icon: (
+			<SVG viewBox="0 0 24 24">
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M20.5875 3.42383L22.0017 4.83804L20.0003 6.83942V18.0006C20.0003 19.1051 19.1049 20.0006 18.0003 20.0006H6.8392L4.83316 22.0066L3.41895 20.5924L20.5875 3.42383ZM12.4198 14.4202L13.4203 15.4207L14.4203 14.4207C15.0499 13.9172 15.9566 13.9645 16.5303 14.5307L18.0003 16.0007V8.83959L12.4198 14.4202ZM6.00035 6.00023H15.1626L17.1626 4.00023H6.00035C4.89578 4.00023 4.00035 4.89566 4.00035 6.00023V17.1625L9.58148 11.5814L8.19035 10.1902L6.00035 12.3802V6.00023Z"
+				/>
+			</SVG>
+		),
+	},
+	[ FEATURE_IMAGE_STORAGE ]: {
+		id: FEATURE_IMAGE_STORAGE,
+		name: translate( 'Image storage' ) as string,
+		description: translate( 'Extended storage space for hi-res images.' ) as string,
+		minSupportedPlan: PLAN_PREMIUM,
+		icon: (
+			<SVG viewBox="0 0 24 24">
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M22 4V16C22 17.105 21.105 18 20 18H7.99999C6.89499 18 5.99999 17.105 5.99999 16V4C5.99999 2.895 6.89499 2 7.99999 2H20C21.105 2 22 2.895 22 4ZM18 19.9998H4V5.99984C2.895 5.99984 2 6.89484 2 7.99984V19.9998C2 21.0998 2.9 21.9998 4 21.9998H16C17.105 21.9998 18 21.1048 18 19.9998ZM16.5 5.99984C15.672 5.99984 15 6.67184 15 7.49984C15 8.32784 15.672 8.99984 16.5 8.99984C17.328 8.99984 18 8.32784 18 7.49984C18 6.67184 17.328 5.99984 16.5 5.99984ZM8 10.3332V4.00016H20V12.2342L19.487 11.6642C18.693 10.7792 17.306 10.7792 16.511 11.6642L15.855 12.3952L11 7.00016L8 10.3332Z"
+				/>
+			</SVG>
+		),
+	},
+	[ FEATURE_VIDEO_STORAGE ]: {
+		id: FEATURE_VIDEO_STORAGE,
+		name: translate( 'Video storage' ) as string,
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **10** times:
`translate( 'No video storage' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-feature-picker)